webapp/sources/rudder/rudder-core/src/main/resources/hooks.d/policy-generation-pre/readme.adoc

```diff
@@ -0,0 +1,14 @@
+= policy-generation-pre
+
+== When/What ?
+
+This directory contains hooks executed before policy generation.
+
+Typically, these hooks are used to check the integrity of techniques.
```

Comment:
You need to explain that these hooks behave differently than all the other ones and that the will PREVENT policy generation on error. 
In other cases, an error just stop the hook pipeline